src/creature.cpp

```diff
@@ -19,34 +19,34 @@
 
 static std::map<int, std::map<body_part, double> > default_hit_weights = {
     {
-        -1, /* attacker smaller */
+        -1, /* attacker smaller (100pts) */
         {   { bp_eyes, 0.f },
             { bp_head, 0.f },
-            { bp_torso, 55.f },
-            { bp_arm_l, 18.f },
-            { bp_arm_r, 18.f },
-            { bp_leg_l, 28.f },
-            { bp_leg_r, 28.f }
+            { bp_torso, 20.f },
+            { bp_arm_l, 15.f },
+            { bp_arm_r, 15.f },
+            { bp_leg_l, 25.f },
+            { bp_leg_r, 25.f }
         }
     },
     {
-        0, /* attacker equal size */
-        {   { bp_eyes, 10.f },
-            { bp_head, 20.f },
-            { bp_torso, 55.f },
-            { bp_arm_l, 28.f },
-            { bp_arm_r, 28.f },
+        0, /* attacker equal size (150pts) */
+        {   { bp_eyes, 0.5f },
+            { bp_head, 3.5f },
+            { bp_torso, 50.f },
+            { bp_arm_l, 30.f },
+            { bp_arm_r, 30.f },
             { bp_leg_l, 18.f },
             { bp_leg_r, 18.f }
         }
     },
     {
-        1, /* attacker larger */
-        {   { bp_eyes, 5.f },
-            { bp_head, 25.f },
-            { bp_torso, 55.f },
-            { bp_arm_l, 28.f },
-            { bp_arm_r, 28.f },
+        1, /* attacker larger (175pts)*/
```

Comment:
If we're using floats anyways why don't we just convert everything over to a flat 100 point scale with the required decimal values? Using different scales on different things just obfuscates the actual percentage values of the numbers; by having everything on a 100 point scale a developer can easily glance at it and see all the percentages at once for each type, rather than needing to pull out their calculator and divide all of the point values in that category by 1.75, all of the ones in that category by 1.5, and so on.